cleanup

```diff
@@ -1,11 +1,36 @@
 #! /usr/bin/env bash
 
 (
+    BROOT="`pwd`"
     cd inst
-    if [ -z $RCAP_BUILD_FAST ]; then
-	npm install
-	bower install --allow-root
-	( cd bower_components/jquery.sparkline && make )
-	grunt
+    if [ -z "$RCAP_BUILD_FAST" ]; then
+	echo Building dependencies ...
+	if npm version | grep npm: >/dev/null; then
+	    echo npm ... OK
+	else
+	    echo 'ERROR: cannot find npm - try "sudo apt-get install npm" first'
+	    exit 1
+	fi
+	if sass -v | grep ^Sass > /dev/null; then
+	    echo Ruby sass ... OK
+	else
+	    echo 'ERROR: cannot find sass - try "sudo apt-get install ruby-sass" first'
+	    exit 1
+	fi
+
+	npm install || exit 1
+
+	## some of the NJS/bower modules are brain-dead and make assumptions about
+	## things being installed system-wide, so we have to use local paths and
+	## add uglifyjs module to the PATH (eek)
+	PATH="$BROOT/inst/node_modules/uglifyjs/bin:$PATH"
+	export PATH
+	node_modules/bower/bin/bower install --allow-root || exit 1
```

Comment:
Does `R CMD build` interpret the exit status? If it does not not stop on a non-zero status, then it might be worth adding some hack to `install.sh` to warn when the "cleanup" script failed.